### ISSUE TITLE
Disconnect from app triggers the error page

### ISSUE THREAD

**Issue by lgrahl** [threema-ch/threema-web#552]
## Expected Behavior

If one closes a session from the app, the welcome page should be displayed along with an alert.

![ok](https://user-images.githubusercontent.com/1793755/43765243-c5a6874e-9a2f-11e8-8ef6-de51a474f1f7.png)

## Current Behavior

If the app closes the web socket connection faster than the web client, an error page is being displayed along with the alert.

![error](https://user-images.githubusercontent.com/1793755/43765226-bdd89f0c-9a2f-11e8-891c-82ab7a09dffd.png)

## Steps to Reproduce (for bugs)

1. Establish a session.
2. Close the session in the app.
3. Hope your web client is slower (or add a short timeout for `this.salty.disconnect()`)

**Comment by dbrgn** [comment]
I can't really reproduce this... Adding an async timeout for `salty.disconnect` isn't a valid way to reproduce this since it may result in disconnecting after the connection has already been re-established on the welcome page.

I added a synchronous busy-loop just before `this.salty.disconnect` that resulted in a 2 second delay. I also tried a synchronous and asynchronous delay in `_receiveConnectionDisconnect` (before this.stop), but none of them resulted in a "connection closed" error.

Maybe it was fixed in the meantime?
